### PR TITLE
Pic 1494 add deploy builds

### PR DESCRIPTION
**Pull request by chris-faulkner-mojdt**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,6 +2,17 @@ version: 2.1
 
 orbs:
   hmpps: ministryofjustice/hmpps@3.2
+  jira: circleci/jira@1.2.2
+
+executors:
+  deployer:
```
Comment (jhevans) [inline .circleci/config.yml:8]: We shouldn't need this I don't think if we're using `hmpps/deploy_env`

File: .circleci/config.yml [hunk 2]
```diff
@@ -25,11 +36,79 @@ jobs:
...
+      - store_artifacts:
+          path: build/reports/tests
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build/libs
+            - build.gradle.kts
+            - app_version.txt
+
+  deploy_dev:
```
Comment (jhevans) [inline .circleci/config.yml:76]: Again these jobs double up the `hmpps/deploy_env` jobs lower down in the workflow

File: .circleci/config.yml [hunk 3]
```diff
@@ -25,11 +36,79 @@ jobs:
       - store_artifacts:
           path: build/reports/tests
 
+  build:
```
Comment (jhevans) [inline .circleci/config.yml:39]: I forget why it is we's not using the `hmpps/validate` job here 🤔  I know there was a reason..

**Review by jhevans — COMMENTED** [review]
Think we have some redundant CI config

File: .circleci/config.yml (hunk 1 above)
Comment (chris-faulkner-mojdt) [reply .circleci/config.yml:8]: Removed

File: .circleci/config.yml (hunk 2 above)
Comment (chris-faulkner-mojdt) [reply .circleci/config.yml:76]: Removed

File: .circleci/config.yml (hunk 3 above)
Comment (chris-faulkner-mojdt) [reply .circleci/config.yml:35]: Perhaps we should use it but it just does test as far as i can see with 

`-Dorg.gradle.jvmargs="--illegal-access=permit" -Dkotlin.daemon.jvm.options="--illegal-access=permit"`

I don't think we need that.

Comment (chris-faulkner-mojdt) [reply .circleci/config.yml:35]: Shall I remove it ?

Comment (jhevans) [reply .circleci/config.yml:35]: Yeah if you don't mind, would be good to use as much stuff as the orb gives us as possible

Comment (jhevans) [reply .circleci/config.yml:35]: Is ours doing anything else other than test?

Comment (chris-faulkner-mojdt) [reply .circleci/config.yml:35]: Our build phase does test and build

**Pull request closed without merging**